test/coralogix.test.js

```diff
@@ -104,6 +105,7 @@ it('CoralogixLogger', async () => {
     // intermittent outages
     await server.listen();
     logger.apiurl = `http://localhost:${server.port}/`;
+    logger.agent = http.globalAgent;
```

Comment:
Let's make private? (just add _ before the name).
This should signal that depending on it is at the discretion of any api users…

Reply:
will do tomorrow.